c7n/resources/redshift.py

```diff
@@ -37,6 +37,20 @@ class Redshift(QueryResourceManager):
     filter_registry = filters
     action_registry = actions
 
+@filters.register('default-vpc')
+class DefaultVpc(DefaultVpcBase):
+    """ Matches if an redshift database is in the default vpc
+    """
+
+    schema = type_schema('default-vpc')
+
+    def __call__(self, redshift):
+        if 'VpcId' in redshift:
```

Comment:
this reduces to a one liner

return redshift.get('VpcId') and self.match(redshift['VpcId']) or False